addons/website_blog/static/src/js/systray_items/blog_post_add_form.js

```diff
@@ -0,0 +1,30 @@
+/** @odoo-module **/
+
+import FormController from 'web.FormController';
+import viewRegistry from 'web.view_registry';
+import FormView from 'web.FormView';
+
+const AddBlogFormController = FormController.extend({
+    saveRecord() {
+        return this._super.apply(this, arguments).then(() => {
+            const state = this.model.get(this.handle);
+            this.do_action({
+                type: 'ir.actions.act_window_close',
+                infos: { path: `${state.data.website_url}?enable_editor=1` },
+            });
+        });
+    },
+});
```

Comment:
@xO-Tx Maybe this part is the same in blog / event / etc, so it could be generic and declared in website (and blog and event would only declare the view using it) (maybe just an URL param since it is a bit different in forum etc).